cmd/stork/stork.go

```diff
@@ -209,6 +209,10 @@ func main() {
 		},
 	}
 
+	// Export USE_GKE_GCLOUD_AUTH_PLUGIN=True
+	log.Warnf("Export USE_GKE_GCLOUD_AUTH_PLUGIN=True")
```

Comment:
What happens if we use this on the oldest supported k8s version GKE cluster? I am assuming it wont cause any problem since this is only being used to authenticate with Google cloud and generate a kubeconfig. We should test it out.

Reply:
I tested with  GCloud CLI version 399, Storkctl from 2.12, GK8s 1.24.9-gke. This created a kubeconfig which doesn't have any reference to gke-gcloud-auth-plugin so it ends up using deprecated gcp auth plugin and it worked.
I think this encironment variable should just be ignored by GCP on older K8s versions. But we can test it out in System Tests.